install.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+
+# VARIABLES, CHANGE AS NEEDED
+GITHUB_REPO="https://github.com/bmFtZQ/edge-frfox.git"
+PROJECT_NAME=$(basename $GITHUB_REPO | cut -d '.' -f 1)
+TMP_DIR="${TMPDIR:-$(dirname $(mktemp))}"
```

Comment:
Is there any reason `mktemp -d` can't be used on its own here to make a new temp directory?

```sh
TMP_DIR="$(mktemp -d)"
```

This seems to work better on macOS.